great_expectations/validator/validator.py

```diff
@@ -360,6 +375,298 @@ def inst_expectation(*args, **kwargs):
 
         return inst_expectation
 
+    def _build_expectation_configuration(
+        self,
+        expectation_type: str,
+        expectation_kwargs: dict,
+        meta: dict,
+        expectation_impl: "Expectation",  # noqa: F821
+    ) -> ExpectationConfiguration:
+        success_keys: Tuple[str] = (
+            expectation_impl.success_keys
+            if hasattr(expectation_impl, "success_keys")
+            else None
+        ) or tuple()
```

Comment:
```suggestion
        success_keys: Tuple[str] = (
            expectation_impl.success_keys
            if hasattr(expectation_impl, "success_keys")
            else tuple()
        )
```
Is this a viable change? If so, please implement throughout.

Reply:
@cdkini -- Done (committed) -- thanks!